packages/notifications-controller/src/constants/constants.ts

```diff
@@ -0,0 +1,13 @@
+export const USER_STORAGE_VERSION = '1';
+
+// Force cast. We don't really care about the type here since we treat it as a unique symbol
+export const USER_STORAGE_VERSION_KEY: unique symbol = 'v' as never;
+
+// TODO - figure out how we can prefill these values
```

Comment:
AHA, nice point! Yes we can pass in the config via the constructor.

@Jonathansoufer This will be needed for Notifications and Push Notifications. If you want, this can be done in a separate PR, but heads up on this :).
- In fact this endpoint will soon be replaced with a custom proxy (but it is not available yet :P)

Reply:
Yeah, let's do it in a subsequence PR, once this will be linked with the change on constant file mentioned in another comment.